drivers/bluetooth/hci/hci_spi_st.c

```diff
@@ -579,16 +595,22 @@ static int bt_spi_open(void)
 	/* Device will let us know when it's ready */
 	k_sem_take(&sem_initialised, K_FOREVER);
 
+#if defined(CONFIG_BT_HCI_RAW) && defined(CONFIG_BT_BLUENRG_ACI)
+	/* force BlueNRG to be on controller mode */
+	uint8_t data = 1;
+
+	bt_spi_send_aci_config(BLUENRG_CONFIG_LL_ONLY_OFFSET, &data, 1);
+#endif /* CONFIG_BT_HCI_RAW && CONFIG_BT_BLUENRG_ACI */
 	return 0;
 }
 
 static const struct bt_hci_driver drv = {
 	.name		= DEVICE_DT_NAME(DT_DRV_INST(0)),
 	.bus		= BT_HCI_DRIVER_BUS_SPI,
-#if defined(CONFIG_BT_BLUENRG_ACI)
 	.quirks		= BT_QUIRK_NO_RESET,
```

Comment:
So `BT_QUIRK_NO_RESET` is now applied for !CONFIG_BT_BLUENRG_ACI configs. 
Does it relate to current change done in this commit ? If yes, can you elaborate in commit message ?

Reply:
Since this driver is only for ST HCI SPI protocol, `BT_QUIRK_NO_RESET` always applies. In other words, as it was inherited from `spi.c`, it should have not been there since the beginning. I will add the note to the commit message.